### DISCUSSION TITLE
Is there a way to add nginx proxy manager to custom locations?

### DISCUSSION THREAD

**Discussion by ytoaa** [NginxProxyManager/nginx-proxy-manager#2962] [Q&A]
<img width="360" alt="화면 캡처 2023-05-31 104942" src="https://github.com/NginxProxyManager/nginx-proxy-manager/assets/24527825/10e03fb5-ccb7-4869-95b6-348e95ef0373">
<img width="363" alt="화면 캡처 2023-05-31 105020" src="https://github.com/NginxProxyManager/nginx-proxy-manager/assets/24527825/f828a5e7-9e21-4139-a7f5-059210a5a461">

If you set it up like this and go to adb.com/nginx/ , you will not be able to connect to nginx.
I think we need to add something to the advance in custom location.
Is there anyone who knows?
For your information, "192.168.0.3:30111" is the address of the nginx proxy manager.

**Comment by jack3308** [comment]
Did you ever solve this? I'm trying to do the exact same right now.

**Comment by komiblog** [comment]
same question

**Comment by spadazz** [comment]
I've tried multiple solutions involving entering parameters in different order in the web ui.
After none of those suggestions was useful to fix this problem, i solved this bypassing the web application altogether.
In my case, I'm using nginxproxymanager inside casaos and the config files are mapped on the path :
/DATA/AppData/nginxproxymanager/data/nginx/proxy_host

Doing an rgrep -i [mydomainname] you can find which proxy config you have to modify.
I needed to create a proxy that : 
from https://wake.[mydomain]
calls  http://[ipaddress]:8089/wolweb/

So i inserted a custom block inside the predefined location /  in the proxy file, like this : 
```
    # custom add 
    if ($uri !~ ^/wolweb/) {
        return 301 /wolweb/;
    }
```
And then restarted the npm app  from casaos.
Now it works.

I suppose that the web forms of the ui aren't working well in updating the proxy configs.

Enrico
